addon/-private/formatters/format-message.ts

```diff
@@ -58,7 +58,9 @@ export default class FormatMessage extends Formatter<Options> {
   // @ts-ignore This class does not match the abstract base class.
   createNativeFormatter = memoize(
     (ast: TranslationAST, locales: string | string[], formatConfig?: Partial<Formats>) => {
-      return new IntlMessageFormat(ast as any, locales, formatConfig);
+      return new IntlMessageFormat(ast as any, locales, formatConfig, {
```

Comment:
@buschtoens curious if you know why `any` is needed since `TranslationAST` is typed to `MessageFormat[]` which is the expected type.

Reply:
So I took a look at this and it seems like these types aren't identical:
```ts
import type { MessageFormatElement } from 'intl-messageformat-parser';
import type { MessageFormatElement } from 'intl-messageformat/intl-messageformat';
// The above is not an actual exported type.
```
They look identical from a quick look but might be an issue with them somewhere.

Reply:
Thanks for investigating @Alonski!

Reply:
Correct! I had that one hit me as well in #1412! But I also didn't look into it deeper yet.

Reply:
We can fix this in a follow-up PR. :)